README.md

```diff
@@ -340,6 +340,7 @@ Examples exist on [codesandbox.io][examples]:
 
 - [downshift Apollo example](https://codesandbox.io/s/j2omZpK3W)
 - [downshift Spectre.css example](https://codesandbox.io/s/M89KQOBRB)
+- [downshift GenieJS example](https://codesandbox.io/s/4zNL2WW1)
```

Comment:
This example is neat, but it's not using the [genie library](https://www.npmjs.com/package/geniejs). Could you update it to use genie?